user-apps/demo-pdf/default.nix

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenv,
+  pkgs,
+  lib,
+  ...
+}:
+stdenv.mkDerivation {
+  name = "demo-pdf";
+
+  src = ./Whitepaper.pdf;
+  phases = ["installPhase"];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ${./Whitepaper.pdf} $out/Whitepaper.pdf
+    echo "zathura $out/Whitepaper.pdf" > $out/bin/run-zathura
```

Comment:
Then if you add `zathura` to the "callPackage-style function call attribute set" above, you can say `${zathura}/bin/zathura` here, instead of zathura.